azurerm/internal/services/digitaltwins/digital_twins_instance_resource.go

```diff
@@ -59,23 +59,25 @@ func resourceArmDigitalTwins() *schema.Resource {
 		},
 	}
 }
-func resourceArmDigitalTwinsCreate(d *schema.ResourceData, meta interface{}) error {
+func resourceArmDigitalTwinsInstanceCreate(d *schema.ResourceData, meta interface{}) error {
 	subscriptionId := meta.(*clients.Client).Account.SubscriptionId
-	client := meta.(*clients.Client).IoTHub.DigitalTwinsClient
+	client := meta.(*clients.Client).DigitalTwins.InstanceClient
 	ctx, cancel := timeouts.ForCreate(meta.(*clients.Client).StopContext, d)
 	defer cancel()
 
 	name := d.Get("name").(string)
 	resourceGroup := d.Get("resource_group_name").(string)
 
+	id := parse.NewDigitalTwinsInstanceID(subscriptionId, resourceGroup, name)
```

Comment:
if we format this here:

```suggestion
	id := parse.NewDigitalTwinsInstanceID(subscriptionId, resourceGroup, name).ID("")
```